components/tables/recent-table.js

```diff
@@ -1,30 +1,20 @@
 import React, {useState} from 'react';
-import styled from 'styled-components';
 import fetch from 'isomorphic-unfetch';
 
-import EditableRow from './editable-row';
-
 import {useFetchUser} from '../../lib/user';
 
-const TableWrapper = styled.div`
-	overflow-x: scroll;
-`;
-
-const StyledTable = styled.table`
-	border: 1px solid black;
+import EditableRow from './editable-row';
+import {StyledRecentTable, StyledRecentTableWrapper} from './styles';
 
-	th,
-	td {
-		padding: 5px;
-		text-align: center;
-	}
-`;
+const RecentTable = ({data, type, account = null}) => {
+	const {user} = useFetchUser();
 
-const RecentTable = ({data}) => {
-	const {user, loading} = useFetchUser();
 	const [isEditing, setIsEditing] = useState(null);
 	const [idBeingEdited, setIdBeingEdited] = useState(null);
-	const [transactionsList, setTransactionsList] = useState(data);
+	const [transactionsList, setTransactionsList] = useState(
+		data.recentTransactions
```

Comment:
Double check that the data coming back from the api for recent and [account] is in the same format and being handled sanely